tests/unittests/cli/utils/test_cli_deploy.py

```
@@ -769,6 +769,31 @@ def test_success_with_app_export(self, tmp_path: Path) -> None:
+    parent_dir = str(tmp_path)
+    if parent_dir not in sys.path:
+      sys.path.insert(0, parent_dir)
+    # Populate sys.path_importer_cache before agent package directory exists
+    for hook in sys.path_hooks:
+      try:
+        finder = hook(parent_dir)
+        if finder and hasattr(finder, 'find_spec'):
+          finder.find_spec("non_existent_module")
+          break
+      except Exception:
+        pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)